src/index.ts

```diff
@@ -477,8 +477,7 @@ export async function trustRemoteMachine(
   _logOrDebug(logger, 'log', `Connecting to remote host ${hostname} via ssh`);
   // Connect to remote box via ssh.
   const child = execa.shell(
-    // @TODO Change this to npx
-    `ssh ${hostname} node devcert/bin/devcert.js remote --port=${port} --cert='${JSON.stringify(
+    `ssh ${hostname} npx @mike-north/devcert-patched remote --port=${port} --cert='${JSON.stringify(
```

Comment:
we need to make absolutely sure that this is _exactly the same version run on remote_ that's being run locally. Please use [pkg-up](https://www.npmjs.com/package/pkg-up) to find the `package.json` file, and read the version number from it
```suggestion
    `ssh ${hostname} npx @mike-north/devcert-patched remote@${version} --port=${port} --cert='${JSON.stringify(
```